src/zoid/buttons/component.jsx

```diff
@@ -877,6 +881,17 @@ export const getButtonsComponent: () => ButtonsComponent = memoize(() => {
           return props?.displayOnly || [];
         },
       },
+
+      message: {
+        type: "object",
+        queryParam: true,
+        required: false,
+        validate: ({ props, value = {} }) => {
```

Comment:
Should we consider using the `decorate` zoid prop option as well similar to `style` which will ensure that the `message` object is formatted correctly and only contains the expected keys?

Reply:
We also likely don't want a default value of `{}` as mentioned in the comment about `default`.

Reply:
noted, will add decorate and remove `{}`